src/config.rs

```diff
@@ -57,6 +57,13 @@ pub fn load(matches: &ArgMatches) -> Config {
         }
     };
 
+    // TODO better way to do this
```

Comment:
We may potentially show a warning here, or shouldn't try to freeze on Windows anyway (even if enabled).

You can leave this as is for now. :+1: